### ISSUE TITLE
Clarification re results in Table 2

### ISSUE THREAD

**Issue by Demfier** [clinc/oos-eval#7]
Hi,

Do the numbers in Table 2 for oos-train correspond to the test set? I am trying to replicate the BERT results (using `transformers` and `datasets` library) with the hyperparameters provided in this repo, but there's a near 10 point difference in the test set performance; however, my validation set performance is quite close to the numbers provided in the paper. It'd be great if you could clarify this.

Thanks,
Gaurav.

**Comment by gxlarson** [comment]
Which column in Table 2? Or are you seeing a 10pt difference across the board?

**Comment by Demfier** [comment]
In-scope accuracy for BERT in oos-train (row 4 from the top). I tried it for full, small, imabal as well as oos+

**Comment by Demfier** [comment]
Hi,

I was able to replicate the results in the paper. So, I am closing this issue. (I was using the incorrect model checkpoint)
